### DISCUSSION TITLE
Navigation bar location change

### DISCUSSION THREAD

**Discussion by DrVektor** [d2phap/ImageGlass#1452] [General]
is possible to change navigation bar location to bottom. I want to control navigation bar bottom becoz its easy to use then top. Please can you tell me how can i change navgationbar location from top to bottom? Thank you and may the easy come. 🧡🙂

**Comment by d2phap** [comment]
You can change it in the Settings > Toolbar:

![image](https://user-images.githubusercontent.com/3154213/203206914-5068f72e-438c-4661-9df8-40b2b565d08b.png)

**Reply by DrVektor** [reply]
Thank you so much. But i am trying to use latest beta development version and it didnt have settings gui like this. It just have text file config but i will search it maybe it can be.

**Reply by d2phap** [reply]
Oh, it just has not supported in v9.0 beta 2

**Reply by DrVektor** [reply]
Thank you :)
